.github/workflows/tests.yml

```diff
@@ -21,32 +21,26 @@ jobs:
     strategy:
       matrix:
           python-version: [3.7, 3.8, 3.9]
-          requirements-level: [min, pypi]
-          db-service: [postgresql11, mysql8]
-
-          exclude:
-          - python-version: 3.9
-            requirements-level: min
+          requirements-level: [pypi]
+          db-service: [postgresql13, mysql8]
 
           include:
-
-          - db-service: postgresql11
+          - db-service: postgresql13
             DB: postgresql
             POSTGRESQL_VERSION: POSTGRESQL_11_LATEST
             SQLALCHEMY_DATABASE_URI: "postgresql+psycopg2://invenio:invenio@localhost:5432/invenio"
-            EXTRAS: "all,postgresql"
```

Comment:
I think `postresql` is needed to be specified in the installed extras, but I might be wrong